spec-meeting/2023/2023-10-12.md

```diff
@@ -0,0 +1,31 @@
+# Spec Meeting 2023-10-12
+
+## On Removing the classic Ember.Object model
+
+ - Application
+	 - extends Namespace.extend(RegsitryProxyMixn)
+	 - customEvents delegation still exists, bubbling changes are potentially complex to manage if we don't provide a good compat story
+ - EmberArray, MutableArray
+	 - Ember.A deprecation would deprecate every feature beyond native array
+	 - an optional feature to disable the Ember.A-enforced mutation of instances would be good. It makes Ember.A a noop, returning a native array.
+ - `@ember/component`
+	 - https://github.com/emberjs/rfcs/pull/389 is merged but unimplemented, we need to do this first
+	 - we need to check if any existing framework-provided components still extend `@ember/component`
```

Comment:
Nope, not in a long time (unless you count engines which still uses the legacy built in components addon, for now)